### ISSUE TITLE
The "Automatically load all model folders" setting doesn't loads the models at start

### ISSUE THREAD

**Issue by menahem121** [comfyanonymous/ComfyUI#5970]
### Expected Behavior

models should be loaded from models folders but none of them are getting automatically loads.

### Actual Behavior

models should be loaded from models folders but none of them are getting automatically loads.

### Steps to Reproduce

simply enable it and verify by yourself

### Debug Logs

```powershell
nothing to show in the logs cause models aren't loading
```

**Comment by menahem121** [comment]
thank you, can i just copy your main.py and replace the one from the official ComfyUI repo?

**Comment by LukeG89** [comment]
The option to load all models folder is only when you open Model Library. If enable, all model are "loaded" (you will see the numbers and you can seach them)

**Comment by menahem121** [comment]
im using docker to install and a script to run comfyU, and if i understand correctly the Model Library is something in the GUI right?

**Comment by LukeG89** [comment]
https://github.com/user-attachments/assets/67048886-30cb-41bc-bb98-345a2e979e11

**Comment by menahem121** [comment]
thank you for your time @LukeG89 sadly its not what im looking for

**Comment by menahem121** [comment]
@mnmnk43434 i don't understand your answer

**Comment by LukeG89** [comment]
> thank you for your time @LukeG89 sadly its not what im looking for

@menahem121 What do you need to do? You don't see your models in the loader nodes?

> @mnmnk43434 i don't understand your answer

It seems that the other account is pretty sketchy...

**Comment by menahem121** [comment]
no i want to load my model from the checkpoint folder automatically to the gpu  when ComfyUI start even if no workflow was run
